database/sql/V020__extend_orders_with_is_liquidity_order.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE orders ADD is_liquidity_order boolean NOT NULL DEFAULT false;
+-- So far every liquidity order was also a partially fillable order and all
```

Comment:
I think this comment may be wrong.

Specifically, we used to allow liquidity orders from stable coin bots with fill or kill orders before introducing partially fillable order support. As such, while all partially fillable orders are liquidity orders, not all liquidity orders are partially fillable orders.

I would reword this to indicate that this provides us with a best guess of what orders were partially fillable, and that other orders are lost to time.